src/Operations.jl

```diff
@@ -1558,7 +1558,9 @@ function stat_rep(x::PackageSpec; name=true)
     if x.repo.rev !== nothing
         rev = occursin(r"\b([a-f0-9]{40})\b", x.repo.rev) ? x.repo.rev[1:7] : x.repo.rev
     end
-    repo = Operations.is_tracking_repo(x) ? "`$(x.repo.source)#$(rev)`" : ""
+    repo = Operations.is_tracking_repo(x) ?
+    string("`$(x.repo.source)", (x.repo.subdir == nothing ? "" : "/[$(x.repo.subdir)]"), "#$(x.repo.rev)`") :
+        ""
```

Comment:
So the old code use `rev`, now you use `x.repo.rev`, leaving `rev` unused. So either `rev` should go, or perhaps this be done? (I don't claim to fully understand that code, so I am guessing):
```suggestion
    repo = Operations.is_tracking_repo(x) ?
    string("`$(x.repo.source)", (x.repo.subdir == nothing ? "" : "/[$(x.repo.subdir)]"), "#$(rev)`") :
        ""
```
And if I might go further, to my personal subjective taste, code would be more readable if you introduce an auxiliary variable:
```suggestion
    subdir = x.repo.subdir == nothing ? "" : "/[$(x.repo.subdir)]"
    repo = Operations.is_tracking_repo(x) ? string("`$(x.repo.source)$(subdir)#$(rev)`") : ""
```

Reply:
Thanks, updated with something like this. And not using `rev` was an error on my part, indeed.